dim.sh

```diff
@@ -15,21 +15,22 @@ curl -L -O $URL
 unzip $ZIP_NAME
 cd $FILE_NAME
 
-# setup.sh is DOS encoded
-tr -d '\015' <setup.sh >setup2.sh 
+# setup.sh is DOS-encoded
+tr -d '\015' < setup.sh > setup2.sh
 mv setup2.sh setup.sh
 
-# compile
-export OS=Linux
-source setup.sh
-gmake realclean
-gmake X64=yes
+# Build (needs special environment)
+( export OS=Linux
+  source setup.sh
+  gmake realclean
+  gmake X64=yes )
 
-# copy to destination
-cp -r dim $INSTALLROOT
-cp -r linux/* $INSTALLROOT  
+# Installation
+rsync -av dim/ $INSTALLROOT/dim/                                                # headers
+rsync -av --exclude "*.o" --exclude "*.so" --exclude "*.a" linux/ $INSTALLROOT/ # executables
+mkdir $INSTALLROOT/lib && cp -v linux/*.{a,so} $INSTALLROOT/lib/                # libraries
```

Comment:
if we have any ways a messy installation folder, I would not move the libs in lib and keep them in the top dir along with the binaries.